README.md

```diff
@@ -119,9 +119,9 @@ For a development installation, you'll need at least:
     - xml
     - json
     - zip
-- MySQL/MariaDB >=5.0
+- MySQL/MariaDB >= 5.6
```

Comment:
Should be `8.0` as per https://github.com/claroline/Distribution/blob/13.x/plugin/cursus/Installation/Migrations/pdo_mysql/Version20200721061805.php#L25. We may rather prefer avoid using 8.0-specific functions though. /cc @Elorfin